backend/src/graphql/resolver/TransactionResolver.test.ts

```diff
@@ -0,0 +1,318 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+import { EventProtocolType } from '@/event/EventProtocolType'
+import { userFactory } from '@/seeds/factory/user'
+import {
+  confirmContribution,
+  createContribution,
+  createUser,
+  sendCoins,
+} from '@/seeds/graphql/mutations'
+import { login } from '@/seeds/graphql/queries'
+import { bobBaumeister } from '@/seeds/users/bob-baumeister'
+import { peterLustig } from '@/seeds/users/peter-lustig'
+import { EventProtocol } from '@entity/EventProtocol'
+import { User } from '@entity/User'
+import { cleanDB, resetToken, testEnvironment } from '@test/helpers'
+import { logger } from '@test/testSetup'
+import { GraphQLError } from 'graphql'
+
+let mutate: any, query: any, con: any
+let testEnv: any
+
+beforeAll(async () => {
+  testEnv = await testEnvironment(logger)
+  mutate = testEnv.mutate
+  query = testEnv.query
+  con = testEnv.con
+  await cleanDB()
+})
+
+afterAll(async () => {
+  await cleanDB()
+  await con.close()
+})
+
+let bobData: any
+let peterData: any
+let user: User[]
+
+describe('send coins', () => {
+  beforeAll(async () => {
+    await userFactory(testEnv, peterLustig)
+    await userFactory(testEnv, bobBaumeister)
+
+    bobData = {
+      email: 'bob@baumeister.de',
+      password: 'Aa12345_',
+      publisherId: 1234,
+    }
+
+    peterData = {
+      email: 'peter@lustig.de',
+      password: 'Aa12345_',
+      publisherId: 1234,
+    }
+
+    user = await User.find({ relations: ['emailContact'] })
+  })
+
+  afterAll(async () => {
+    await cleanDB()
+  })
+
+  describe('wrong recipient', () => {
+    it('unknown recipient', async () => {
+      await mutate({
+        query: login,
+        variables: bobData,
+      })
+      expect(
+        await mutate({
+          mutation: sendCoins,
+          variables: {
+            email: 'wrong@email.com',
+            amount: 100,
+            memo: 'test',
+          },
+        }),
+      ).toEqual(
+        expect.objectContaining({
+          errors: [new GraphQLError('No user with this credentials')],
+        }),
+      )
+    })
+
+    it('deleted recipient', async () => {
+      // delete bob
+      const bob = await User.findOneOrFail({ id: user[1].id })
```

Comment:
You could seed Stephen Hawking. This is a deleted user

Reply:
hahahaha, nasty humor, didn't know that :p 
implemented like that now